src/Illuminate/Console/Scheduling/ScheduleListCommand.php

```diff
@@ -220,7 +220,15 @@ private function getClosureLocation(CallbackEvent $event)
             );
         }
 
+        if (is_string($callback)) {
+            return $callback;
+        }
+
         if (is_array($callback)) {
+            if (is_string($callback[0])) {
```

Comment:
Perhaps rather than repeating `sprintf` we could extract a variable?

```php
if (is_array($callback)) {
  $className = is_string($callback[0]) ? $callback[0] : $callback[0]::class;

  return sprintf('%s::%s', $className, $callback[1]);
}
```

Reply:
Good point @lukeraymonddowning! - Change made now 👍